jupyter-js-widgets/src/utils.js

```diff
@@ -141,24 +140,38 @@ function reject(message, log) {
  * text: option string
  */
 function typeset(element, text) {
-    var $el = element.jquery ? element : $(element);
-    if(arguments.length > 1){
-        $el.text(text);
+    if (arguments.length > 1) {
+        if (element.length) {
+            for (var i = 0; i < element.length; ++i) {
+                var el = element[i];
+                el.textContent = text;
+            }
+        } else {
+            element.textContent = text;
+        }
     }
-    if(!window.MathJax){
-        return;
```

Comment:
In a separate PR we should check `typeof window === 'undefined' || !window.MathJax`